compiler/include/symbol.h

```diff
@@ -583,6 +589,9 @@ const char* intentDescrString(IntentTag intent);
 // pass-by-reference intents are used.
 bool argMustUseCPtr(Type* t);
 
+void substituteVarargTupleRefs(BlockStmt* ast, int numArgs, ArgSymbol* formal,
```

Comment:
@noakesmichael - symbol.cpp does not #include resolution.h. So following your suggestion prevents building the compiler.

How do you suggest resolving this issue and how important is it to you vs. leaving substituteVarargTupleRefs() in symbol.h ?